src/niscope/system_tests/test_system_niscope.py

```diff
@@ -21,9 +21,8 @@
 # TODO(sbethur): Use `get_channel_names` when #1402 is fixed
```

Comment:
There's this comment about using get_channel_names (once we've added it, which we have).

Until this week it was simply
>\# TODO(sbethur): Use `get_channel_names` when #1402 is fixed
>test_channels = 'FakeDevice2/0,FakeDevice1/1'

I'd rather specify the values explicitly than rely on the output of a function being tested here. It's clearer what we're testing and the behavior won't change if we mess up get_channel_names.

Reply:
I suppose, the argument is that using `get_channel_names` is good practice for client code but I'm ok either way.

Reply:
Removed the TODO.